app/routes/contact/components/ContactForm.js

```diff
@@ -57,14 +57,22 @@ const ContactForm = (props: Props) => {
     <Form onSubmit={props.handleSubmit(submit)}>
       <p>
         Dette skjemaet kan benyttes for å kontakte Abakus sine komiteer eller
-        Hovedstyret. Sender du til en spesifikk komité er det kun lederen av
+        Hovedstyret. Dersom du har opplevd avvik eller ubehagelige hendelser i
+        forbindelse med Abakus, ønsker vi gjerne at dette meldes fra om her.
+        Send oss gjerne også spørsmål, feedback eller ting du ønsker å fortelle
+        oss. Du kan også benytte dette skjemaet om du ønsker å avtale tid for
+        innsyn i HSP-avtalen.
+      </p>
+      <p>
+        Sender du meldingen til en spesifikk komité er det kun lederen av
         komiteen som vil motta meldingen. Om du sender til Hovedstyret vil hele
-        styret få meldingen.
+        styret motta meldingen.
       </p>
       <p>
-        Send oss gjerne spørsmål, feedback eller ting du ønsker å fortelle oss.
-        Du kan også benytte dette skjemaet om du ønsker å avtale tid for innsyn
-        i HSP-avtalen.
+        Skjemaet kan både benyttes til å sende inn en anonym henvendelse, eller
```

Comment:
Also, now that I think about it, should we explain how the message is sent anonymously? And how the receiver actually receives it. I would image that would increase trust and make it easier for people to reach out!

Reply:
I think we should leave any technical explanation out, if that's what you meant. In my opinion it would just make the text longer and more confusing for most users who probably just care about being anonymous, not how we do it.

Reply:
Yes, wouldn't want to make it longer than needed. But, maybe a tooltip could be nice? Or a footnote at the bottom hidden away with a tooltip. The tooltip would only require an extra sentence. Sort of like the tooltip in the corner on pools (by the question mark). 

> most users who probably just care about being anonymous, not how we do it.

Maybe, but, I know that I personally would be extremely skeptical of a system like abakus.no made by a bunch of students, especially if I was a first grader fresh out of buddy week. It was just a thought.

Reply:
A tooltip could actually be nice, good idea! I'll see what I can do:)

Reply:
When i think about it, I have to catch a flight tomorrow requiring me to be up in 3 hours, so i should probably get some sleep. I'll make the other changes, and then add the tooltip when I have some free time on vacation (as adding an explanatory sentence about how we handle anonymity would actually require me to look into how we handle anonymity first hehe)